### ISSUE TITLE
Support interlace in h.264 decoder

### ISSUE THREAD

**Issue by svitvitskiy** [jcodec/jcodec#241]
Some users (see issue #199) need to decode interlaced videos.

**Comment by maxbrito500** [comment]
Hello, any progress on this ticket?

I'm still getting "org.jcodec.api.UnsupportedFormatException: MPEG PS is temporarily unsupported."

Thanks.

**Comment by svitvitskiy** [comment]
Yeah, I am sorry, probably not going to implement that, unless there’s a
serios contracting opportunity.
